app/serializers/export/contact_serializer.rb

```diff
@@ -1,12 +1,28 @@
 module Export
   class ContactSerializer < ActiveModel::Serializer
     attributes :sharing,
-               :receiving,
+               :following,
```

Comment:
When one sets `receiving` to true, one could see the private profile of the person (when it's already on the pod for another contact), even when the person never shared the profile with the importing user. And the only way we can be sure one is allowed to see the private profile of a person is, when we ask the person, and not trust the person who says "I'm allowed to see that, trust me!"

So we need it in the export, so we can send the `account_migration` to them, but we should import it with `receiving` set to `false` until the person sends us a `contact` message again, as response to the `account_migration` message.